packages/schemas/tables/captcha_providers.sql

```diff
@@ -0,0 +1,12 @@
+create table captcha_providers (
```

Comment:
Better have a userfriendly 'name' column? 

P.S. 
Can a tenant have multiple CaptchProvider entities with the same provider? 
If a tenant has multiple CaptchProvider records, which one would be used in the sign-in experience?

Reply:
we don't need "name", because this can not be customized. And also no metadata storage on the database.

All the metadata including name will be fixed in the codebase with a "provider schema".

Multiple captcha provider is not allowed at this moment.

Reply:
Our goal is simplicity and effectiveness.

Reply:
We typically enforce this constraint through our code logic, for example, in the `sign_in_experience` table.